base/applications/rapps/appdb.cpp

```diff
@@ -34,16 +34,16 @@ CAppDB::CAppDB(const CStringW &path) : m_BasePath(path)
     m_BasePath.Canonicalize();
 }
 
-CAppInfo *
-CAppDB::FindByPackageName(const CStringW &name)
+CAvailableApplicationInfo *
```

Comment:
There is no change to the FindByPackageName method other than that its implementation is now simply `CAppInfo *     FindByPackageName(const CStringW &name) { return FindAvailableByPackageName(name); }` (the old code only returned CAvailableApplicationInfo entries, never CInstalledApplicationInfo). The new method FindAvailableByPackageName is needed by the /uninstall switch so it can read the DB_REGNAME db entry so it can find the registry key name, also used by the new installer code for the same reason, access to the db.